src/Quasar/QuasarF.purs

```diff
@@ -56,6 +56,8 @@ data QuasarF a
   | GetMount AnyPath (MountConfig :~> a)
   | CreateMount AnyPath MountConfig (Maybe Seconds) (Unit :~> a)
   | UpdateMount AnyPath MountConfig (Maybe Seconds) (Unit :~> a)
+  | MoveMount AnyPath AnyPath (Unit :~> a)
```

Comment:
`MountF Move` where `type Move a = { from :: a, to :: a}` so we are extremely safe :D

Reply:
https://github.com/slamdata/purescript-quasar/pull/104